components/Footer.js

```diff
@@ -30,7 +29,11 @@ const Footer = () => {
 										title={link.title}
 										className='fill-current text-acm-white hover:text-acm-blue'
 									>
-										{link.image}
+										{(function () {
```

Comment:
💀 what is this 💀

Reply:
OH LMAO  ITS DEBUG CODE XDDDDDDDDDDDDDDD
i was trying to see if i could modify a style after the component was called but before it was inserted (it didnt work)